src/test/input.ts

```diff
@@ -0,0 +1,42 @@
+export const user1 = `#graphql
+mutation {
+    createUser (data: {
+      name: "eu",
+      email: "eu@gmail.com",
+      birthDate: "27/12/1900",
+      password: "mudar123"
+    } 
+  ) {
+      birthDate
+      email
+      id
+      name
+  }
+}`;
+
+export const queryUser = `#graphql
+query {
+    findUser (id: 1) {
+      birthDate
+      email
+      name
+      hash
+  }
+}`;
+
+export const userDatabase = {
+  data: {
+    findUser: { birthDate: '27/12/1900', email: 'eu@gmail.com', name: 'eu', hash: '7f658385f4d54cba7ce3' },
+  },
+};
+
+export const expectedResponse = {
```

Comment:
melhor este `expectedResponse` ficar no `it` que está sendo utilizado mesmo, para dar maior contexto pra quem lê (lendo nesse arquivo eu não entenderia o contexto, precisaria ir la no teste pra entender)